gsy_framework/sim_results/aggregate_results.py

```diff
@@ -144,16 +151,15 @@ def generate(self):
 
         sorted_market_timeslots = sorted(self.bids_offers_trades)
         start_time = sorted_market_timeslots[0]
-        next_time = start_time + self.resolution
+        next_time = start_time + self.resolution - self.simulation_slot_length
```

Comment:
I am actually having a hard time to find a name instead of simulation_slot_length in order to not correlate the term slot length (which is market-related) to these calculations (which are timeseries - related). Maybe datapoint_interval or datapoint_frequency could work but I also do not like them since they are generic.